03.bowling/bowling.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'rubygems'
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+last_frame_shot = 18
```

Comment:
```suggestion
LAST_FRAME_SHOT = 18
```

どんな状況でも同じ値は定数にしましょう。
あと、定数はプログラムの先頭に記述されることが多いです。

Reply:
定数が頭を大文字とすることでなることを失念していました。修正します。